graphql_query_examples/graphql_mutation_examples.md

```diff
@@ -54,6 +54,108 @@ mutation aliasDelete_for_a_service{
 }
 ```
 
+### 🧬 awsIntegrationUpdate, get AWS integrations first, then reactivate integration for next scheduled sync
+
+Note: `integrationReactivate` will reactivate the integration and OpsLevel will retry syncing resources on the next scheduled sync.
```

Comment:
maybe worth mentioning that this is only needed if the integration was invalidated (per the `invalidatedAt` field)?

Reply:
Updated to call this out!